test_app/management/commands/create_demo_data.py

```diff
@@ -0,0 +1,30 @@
+from crum import impersonate
+from django.core.management.base import BaseCommand
+
+from test_app.models import EncryptionModel, Organization, Team, User
+
+
+class Command(BaseCommand):
+    help = 'Creates demo data for development.'
+
+    def handle(self, *args, **kwargs):
+        if Organization.objects.exists():
+            print('It looks like you already have data loaded, great!')
+            return
```

Comment:
Instead of this do you want to just do `update_or_create` calls below?

Reply:
I kind of don't want to do it because it creates more work about figuring out what lookup fields are. I want to keep this scrappy right now.